types/react-color/react-color-tests.tsx

```diff
@@ -35,7 +38,7 @@ const CustomComponent: FunctionComponent<CustomProps> = (props: CustomProps) =>
             <Alpha color={props.color} onChange={onChange} />
             <Checkboard size={10} white="transparent" grey="#333" />
             <EditableInput value={props.color} label="Test" onChange={onChange} />
-            <Hue color={props.color} direction="horizontal" onChange={onChange} />
+            <Hue color={props.color} direction="horizontal" onChange={onChange} pointer={customPointer} />
```

Comment:
I think it'd probably be worthwhile to leave the original line just to test that it works without the prop.

Reply:
Oh I missed this comment; should I make a new PR or does the system support updating an existing PR?

Reply:
This is merged so can't be modified; you'd need a new PR.

Reply:
Alright, thanks!